packages/api/tests/lambdas/test-db-indexer.js

```diff
@@ -224,196 +129,43 @@ test('getTableName() returns the full name of the DynamoDB table associated with
 
 test('getTableIndexDetails() returns undefined for unsupported table', (t) => {
   t.is(getTableIndexDetails('fake-table-name'), undefined);
-  t.is(getTableIndexDetails(GranuleFilesCache.cacheTableName()), undefined);
+  t.is(getTableIndexDetails('foobar'), undefined);
```

Comment:
We technically don't need a second verification of `getTableIndexDetails`, right? The assertion on line 131 seems sufficient?

Reply:
322ee8324d

Reply:
322ee8324d